### PR TITLE
Add SharingReport

### PR DESCRIPTION
**Pull request by salarson**

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by salarson** [comment]
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/450

**Comment by salarson** [comment]
> the article for the cmdlet is here https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/blob/main/sharepoint/sharepoint-ps/sharepoint-online/New-SPOSiteSharingReportJob but it lacks the .md extension
> 
> Can you please add to this PR or create a new PR to fix the extension? @salarson

https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/450

**Review by get-itips — APPROVED** [review]

**Comment by salarson** [comment]
> Could you please fill out the missing descriptions?

Sorry, but where exactly?

File: sharepoint/sharepoint-ps/sharepoint-online/sharepoint-online.md [hunk 1]
```diff
@@ -307,6 +307,10 @@ The following cmdlet references are for SharePoint Online.
 
 {{Manually Enter New-SPOSiteGroup Description Here}}
 
+### [New-SPOSiteSharingReportJob](New-SPOSiteSharingReportJob.md)
+
+This cmdlet is not currently active in Production and may be removed in the future. You will receive the error, "Site collection sharing report feature has not been enabled", if you run this cmdlet which is currently by design.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/sharepoint-online.md:312]:
```suggestion
This cmdlet is not currently active in production and may be removed in the future. You will receive the error "Site collection sharing report feature has not been enabled" if you run this cmdlet, which is by design. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**